.github/workflows/build_and_test.yml

```diff
@@ -770,7 +790,20 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: [ v14, v15, v16, v17 ]
+        version:
+          # Much data was already generated on old PG versions with bullseye's
+          # libraries, the locales of which can cause data incompatibilities.
+          # However, new PG versions should check if they can be built on newer
```

Comment:
we don't just *check* here, we also create the compute image, right?

Reply:
Updated the comment, thanks!